coreapi/views.py

```diff
@@ -179,15 +179,32 @@ def post(self, request, *args, **kwargs):
             return Response(feedback_serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
 
+import matplotlib.pyplot as plt
```

Comment:
same with this file, coreapi/main_api.py module is enough. We are also not using views module anymore.